src/config/provider/hashicorp.rs

```diff
@@ -0,0 +1,21 @@
+//! Configuration for HashiCorp Vault
+
+use crate::chain;
+use serde::Deserialize;
+
+#[derive(Clone, Deserialize, Debug)]
+#[serde(deny_unknown_fields)]
+/// Hashicorp Vault signer configuration
+pub struct HashiCorpConfig {
```

Comment:
It would be nice to let the `access_token` to be read from the file, for instance:
https://github.com/iqlusioninc/tmkms/blob/ed24642d950f87da7f20ea9b031a6671b9021b8c/src/config/provider/yubihsm.rs#L57